src/50_Sample_AMPs/eCommerce_Sample.html

```diff
@@ -0,0 +1,86 @@
+<!---{
+    "preview": true
+  }--->
+<!--
+  #### Introduction
+
+  Sample page showing item to sell
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+    amp-accordion and amp-carousel
+  -->
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  
+  <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
+  <script async custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js"></script>
+
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    The sample page showing item 6P nexus device to sell
+  -->
+
+    <h1>Nexus 6P</h1>
+
+    <amp-carousel height="300" layout="fixed-height" type="carousel">
+    <amp-img src="/img/all_nexuses.png" width="400" height="300"></amp-img>
+    <amp-img src="/img/gold_nexus.png" width="400" height="300"></amp-img>
+    <amp-img src="/img/silver_nexus.png" width="400" height="300"></amp-img>
+  </amp-carousel>
+
+  <h4>A 5.7-inch, high-resolution WQHD AMOLED display and front-facing stereo speakers — all housed in a contoured aluminum body.</h4>
+
+
+  <button on="tap:amp-user-notification1.dismiss">Add to basket</button>
```

Comment:
Is this button supposed to disappear when clicking? I am running the page locally, when I click on it nothing happens.